src/scripts/main.js

```diff
@@ -1,7 +1,22 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  notification.classList.add('notification', type);
+  const titleElement = document.createElement('h2');
+  titleElement.classList.add('title');
+  titleElement.textContent = title;
+  notification.appendChild(titleElement);
+  const descriptionElement = document.createElement('p');
+  descriptionElement.classList.add('description');
+  descriptionElement.textContent = description;
+  notification.appendChild(descriptionElement);
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+  document.body.appendChild(notification);
+  setTimeout(() => {
+    notification.style.visibility = 'hidden';
```

Comment:
The current implementation sets the notification's visibility to 'hidden' after 2 seconds. This will make the notification invisible but still present in the DOM. Consider removing the notification from the DOM entirely using `notification.remove()` for better performance and to prevent unnecessary elements from accumulating in the DOM.